text/0000-slatepack.md

```diff
@@ -0,0 +1,384 @@
+  - Example `SlatePack Address`: `grin1p4fuklglxqsgg602hu4c4jl4aunu5tynyf4lkg96ezh3jefzpy6swshp5x`
+
+6. _Sending a mobile Grin transaction should be as easy as scanning a simple QR encoded from a bech32 `SlatePack Address`_
+  - Or as easy as pasting the `SlatePack Address` of your counterparty into your wallet for any other device
+  - Or in the worst case, if Tor is not working or blocked for one of the parties, as easy as copying and pasting a couple of `SlatePack Message` strings with a counterparty in an alternative communication channel (email, forum, social media, instant messenger, generic web text box, carrier pigeon etc.)
+
+# Reference-level explanation
+[reference-level-explanation]: #reference-level-explanation
+
+The SlatePack standard defines the three primary components: `SlatePack Address`, `SlatePack Message` and `SlatePack Workflow`.
+
+The `SlatePack Address` is a shareable bech32 encoded ed25519 public key that can be used both to _route synchronous transactions_ and to _encrypt asynchronous transactions_.
+
+The `SlatePack Message` is an easily copy and pastable ascii-armor string that contains an encrypted slate payload by default and is used in asynchronous transactions.
+
+The `SlatePack Workflow` specifies how both of these components interact in a universally adoptable transaction standard for Grin.
+
+## `SlatePack Address`
+
+A `SlatePack Address` is a bech32 encoded ed25519 public key and when shared with other parties is used to represent the ability to receive Grin transactions.
+
+grin-wallet already handles ed25519 keys for the v3 onion addresses in Tor transactions. These keys can be extended to be a general `SlatePack Address` to allow a universal key format for both transport and encryption that is error-checked, QR friendly and easily human identifiable.
+
+  - Existing ed25519 public keys from the wallet are bech32 encoded with `grin` as the `human-readable part` to build a `SlatePack Address`
+
+  - A `SlatePack Address` can be decoded to its ed25519 public key which can then be mapped to an x25519 public key to be used for encryption
+
+Wallets should support the ability for a user to both use a static, reusable `SlatePack Address` or to generate a unique `SlatePack Address` for each transaction if they wish.
+
+ed25519 keys are bech32 encoded as `SlatePack Addresses` rather than x25519 keys because the mapping from ed25519 to x25519 is more straight forward (x25519 public keys do not carry a `v` coordinate so they can map to two possible ed25519 public keys- this is solvable but using the ed25519 as the first order key avoids a potentially complex solution).
+
+#### Key Generation
+
+_The derivation strategy for ed25519 keys used for a `SlatePack Address` in the context of grin-wallet remains open_
+
+#### Example `SlatePack Address`
+
+```
+grin1p4fuklglxqsgg602hu4c4jl4aunu5tynyf4lkg96ezh3jefzpy6swshp5x
```

Comment:
As discussed, perhaps makes sense to stick with slatepack branding on the addresses throughout. 

Also: we may want to have different address prefixes for mainnet and floonet.

Reply:
Updated but leaving this comment open to make sure we don't forget to adjust prefixes for mainnet/floonet once we converge on HRP branding for addresses (slatepack, spk, sp, grin, g etc.)